src/osa/configs/sequencer.cfg

```diff
@@ -71,6 +72,7 @@ PARTITION_PEDCALIB: short, long
 PARTITION_DATA: short, long
 MEMSIZE_PEDCALIB: 3GB
 MEMSIZE_DATA: 16GB
+MEMSIZE_GAINSEL: 15GB
```

Comment:
even if this is a dummy cfg, in the actual cfg in use, this mem req can be reduced, now that the memory leak was fixed in protozfit